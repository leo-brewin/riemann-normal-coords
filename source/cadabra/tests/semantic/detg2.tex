\documentclass[12pt]{cdblatex}

\begin{document}

\section*{\jobname}

\CdbSetup{action=hide}

\begin{cadabra}
   import shared

   import cdblib

   term00A = cdblib.get ('check000','expected/detg2.json')
   term01A = cdblib.get ('check001','expected/detg2.json')
   term02A = cdblib.get ('check002','expected/detg2.json')

   term00B = cdblib.get ('check000','output/detg2.json')
   term01B = cdblib.get ('check001','output/detg2.json')
   term02B = cdblib.get ('check002','output/detg2.json')

   diff000 = shared.check (term00A,term00B)   # cdb (diff000,diff000)
   diff001 = shared.check (term01A,term01B)   # cdb (diff001,diff001)
   diff002 = shared.check (term02A,term02B)   # cdb (diff002,diff002)

\end{cadabra}

\begin{dgroup*}
   \Dmath*{ \cdb*{diff000} }
   \Dmath*{ \cdb*{diff001} }
   \Dmath*{ \cdb*{diff002} }
\end{dgroup*}

\end{document}
