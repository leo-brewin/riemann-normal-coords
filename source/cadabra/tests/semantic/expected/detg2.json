{
  "check000": "1 -  1/3 x^{a} x^{b} R_{a b} -  1/6 x^{a} x^{b} x^{c} \\nabla_{a}(R_{b c}) +  1/180 x^{a} x^{b} x^{c} x^{d} (-9\\nabla_{a b}(R_{c d}) + 10R_{a b} R_{c d}-2g^{e f} g^{g h} R_{a e b g} R_{c f d h}) +  1/90 x^{a} x^{b} x^{c} x^{d} x^{e} (-\\nabla_{a b c}(R_{d e}) + 5R_{a b} \\nabla_{c}(R_{d e})-g^{f g} g^{h i} R_{a f b h} \\nabla_{c}(R_{d g e i}))",
  "check001": "1 -  1/6 x^{a} x^{b} R_{a b} -  1/12 x^{a} x^{b} x^{c} \\nabla_{a}(R_{b c}) +  1/360 x^{a} x^{b} x^{c} x^{d} (-9\\nabla_{a b}(R_{c d}) + 5R_{a b} R_{c d}-2g^{e f} g^{g h} R_{a e b g} R_{c f d h}) +  1/360 x^{a} x^{b} x^{c} x^{d} x^{e} (-2\\nabla_{a b c}(R_{d e}) + 5R_{a b} \\nabla_{c}(R_{d e})-2g^{f g} g^{h i} R_{a f b h} \\nabla_{c}(R_{d g e i}))",
  "check002": " -  1/3 x^{a} x^{b} R_{a b} -  1/6 x^{a} x^{b} x^{c} \\nabla_{a}(R_{b c}) +  1/180 x^{a} x^{b} x^{c} x^{d} (-9\\nabla_{a b}(R_{c d})-2g^{e f} g^{g h} R_{a e b g} R_{c f d h}) +  1/90 x^{a} x^{b} x^{c} x^{d} x^{e} (-\\nabla_{a b c}(R_{d e})-g^{f g} g^{h i} R_{a f b h} \\nabla_{c}(R_{d g e i}))"
}
